lib/gui/app.js

```diff
@@ -118,7 +118,7 @@ app.run((ErrorService) => {
     includeUnstableChannel
   });
 
-  s3Packages.getLatestVersion(currentReleaseType, {
+  githubReleases.getLatestVersion({
```

Comment:
I changed the semantics a bit to accommodate to `electron-builder`'s auto update functionality, which allows the user to move between unstable and production channels. Now, when determining the available versions, we'll query ALL of them, but filter the pre-release ones if `includeUnstableChannel` is not enabled.